tests/Models/UserWithEmbeds.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests\Models;
+
+class UserWithEmbeds extends User
+{
+    protected $withEmbeds = ['addresses'];
```

Comment:
Having this new configuration to "fix" the bugged behavior of the embed relationship seems wrong.

The default `$with` property can be leveraged:

```suggestion
    protected $with = ['addresses'];
```